### PR TITLE
Add Sample Motor Data Limerick

### PR DESCRIPTION
**Pull request by asyatrhl**

### CODE REVIEW

**Review by rotx-eva — APPROVED** [review]

**Review by seldauyanik-maxim — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
...
+- Heavy Ball Bearing Fault
+
+For imbalance faults, varying loads are applied in the setup, a sample can be seen here:
+
+<img src="resources/SampleMotorDataLimerick_sampleImbalance.png" alt="Sample Imbalance" style="zoom:33%;" />
+
+Details for other fault types:
+<img src="resources/SampleMotorDataLimerick_otherFaults.png" alt="Other Faults" style="zoom: 33%;" />
+
+Each fault was tested at 600, 1200, 1800, 2400 and 3000 RPM and tested both with and without an additional 11 lb (5 kg) load on the shaft. ADXL356 sensor data is used for vibration raw data.
```
Comment (ermanok) [inline README.md]: "ADXL356 sensor data is used for vibration raw data" might be rephrased like "ADI's ADXL356 Tri-Axis MEMS Accelerometer is used to collect the motor vibration data."

**Review by ermanok — COMMENTED** [review]
Looks good. Just added one comment..

File: README.md (hunk 1 above)
Comment (asyatrhl) [reply README.md]: Updated.

**Review by rotx-eva — APPROVED** [review]

**Review by vicloginov — APPROVED** [review]
Looks good

**Review by aniktash — COMMENTED** [review]
Looks good

**Pull request merged**